paymentsheet/src/main/java/com/stripe/android/common/model/CommonConfiguration.kt

```diff
@@ -9,6 +10,9 @@ import com.stripe.android.paymentsheet.PaymentSheet
 import com.stripe.android.paymentsheet.addresselement.AddressDetails
 import kotlinx.parcelize.Parcelize
 
+@RestrictTo(RestrictTo.Scope.LIBRARY)
+const val IsEKClientSecretValidRegexPattern = "^ek_[^_](.)+$"
```

Comment:
can this be a private (or internal) variable rather than using restrict to? It doesn't look like it's being used outside this file